arm/Microsoft.ApiManagement/service/apis/deploy.bicep

```diff
@@ -107,7 +107,7 @@ resource apiVersionSetResource 'Microsoft.ApiManagement/service/apiVersionSets@2
 }
 
 resource apis 'Microsoft.ApiManagement/service/apis@2020-06-01-preview' = {
```

Comment:
Let's update this to this cool model Erika & Margarita implemented. It looks like this:

```bicep
resource apiManagementService 'Microsoft.ApiManagement/service@2020-12-01' = existing {
  name: apiManagementServiceName
}

resource apiVersionSetResource 'Microsoft.ApiManagement/service/apiVersionSets@2020-06-01-preview' = if (!empty(apiVersionSet)) {
  name: apiVersionSetName
  properties: apiVersionSet.properties
  parent: apiManagementService 
}

resource apis 'Microsoft.ApiManagement/service/apis@2020-06-01-preview' = {
  name: name
  parent: apiManagementService 
  properties: {
    apiRevision: !empty(apiRevision) ? apiRevision : null
    apiRevisionDescription: !empty(apiRevisionDescription) ? apiRevisionDescription : null
```

Reply:
Ideally for the other child-resources too.

Reply:
done